docs/dev-fork.md

```diff
@@ -0,0 +1,69 @@
+## Fork test / development
+
+!NOTES!
+You need an Infura with [Archived Data](https://infura.io/docs/ethereum/add-ons/archiveData) access
```

Comment:
Either `an Infura node` or `Infura` would be more correct
The same is for Alchemy